src/main/java/org/eolang/sys/Glossary.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2021-2022 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.sys;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import org.eolang.ExFailure;
+
+/**
+ * Known system functions.
+ * @since 0.1
+ */
+public final class Glossary {
```

Comment:
@mximp why it's `public`?

Reply:
@yegor256 We need some entry point to select a function to call from clients (`EOcall` in this case). What would it be if not `Glossary`?

Reply:
@mximp but nobody will use `EOcall` directly. user will use `QQ.sys.call` EO object. everything that's inside it -- is hidden

Reply:
@yegor256 That's right... And `QQ.sys.call` refers to `EOcall` java class sitting in `EOorg.EOeolang.EOsys` package. And it needs an access to `Glossary` which is another `org.eolang.sys` package. 
So `Glossary` cannot be package-private.

Reply:
@mximp why Glossary is in a different package?

Reply:
@yegor256 To fit into common package structure: my understanding was that `EOorg.EOeolang.*` packages contained classes which represents/bounds to EO objects defined in `.eo` files, whereas `org.eolang.*` packages had all supplementary functionality which is not directly bound to EO objects.

Reply:
@yegor256 what do you think about above?